Bottom: ae2aa111e47cdf935b9da772f0ae2e7480f30ce2
Top:    fbbf0c77eeb53423614e242d2f1718224f732bf4
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 17:37:45 -0700

Refresh of fixup

---
diff --git a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
index bcb8d46b30a..8b5539feb1c 100644
--- a/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
+++ b/lldb/packages/Python/lldbsuite/test/python_api/file_handle/TestFileHandle.py
@@ -747,7 +747,7 @@ class FileHandleTestCase(lldbtest.TestBase):
         with open(self.out_filename, 'w') as f:
             status = debugger.SetOutputFile(f)
             self.assertTrue(status.Success())
-            self.handleCmd(debugger, "help help", collect_result=False)
+            self.handleCmd("help help", check=False, collect_result=False)
             # make sure the file wasn't closed early.
             f.write("\nZAP\n")
             lldb.SBDebugger.Destroy(debugger)
